src/mixins/highlightConfig.js

```diff
@@ -55,49 +58,42 @@ export default {
   methods: {
     setHighlight() {
       if (this.highlighted) {
-        this.shapeView.highlight(this.shapeBody, {highlighter: defaultHighlighter});
+        this.shapeView.highlight(this.shapeBody, { highlighter: defaultHighlighter });
         return;
       }
 
-      this.shapeView.unhighlight(this.shapeBody, {highlighter: defaultHighlighter});
+      this.shapeView.unhighlight(this.shapeBody, { highlighter: defaultHighlighter });
     },
     setErrorHighlight() {
       if (!this.shapeView) {
         return;
       }
-
-      if (this.hasError) {
+      if (this.hasError && this.autoValidate) {
         this.shapeView.highlight(null, { highlighter: errorHighlighter });
-      } else {
-        this.shapeView.unhighlight(null, { highlighter: errorHighlighter });
+        return;
       }
+      this.shapeView.unhighlight(null, { highlighter: errorHighlighter });
     },
-    configureCrown() {
-      this.paper.once('render:done', () => this.setHighlight());
```

Comment:
This callback was unnecessary since the mounted method checks if rendering is still happening before calling this method.

Reply:
I'm seeing that the highlight is now a square. Any ideas which change caused it?